core/src/main/java/com/amplifyframework/core/model/AuthStrategy.java

```diff
@@ -25,23 +27,51 @@
      * Owner authorization specifies whether a user can access or operate against an object.  To use OWNER, the API
      * must have Cognito User Pool configured.
      */
-    OWNER,
+    OWNER(AuthRuleProvider.AMAZON_COGNITO_USER_POOLS, List.of(AuthRuleProvider.AMAZON_COGNITO_USER_POOLS,
```

Comment:
Hm. Is there desugaring for `List.of`? That's a pretty modern construct.

Reply:
It didn't bark, so I figured it was OK 🤣  I'll verify though.

Reply:
Take that back...I just got bit by it.